misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -34,79 +34,70 @@ abstract class PathAssertionsSpec(
+
+        context("folder exists") {
+            it("${exists.name} does not throw") {
                 val file = tempFolder.newFolder("test")
                 expect(file).existsFun()
             }
-        }
-    }
 
-    describeFun(existsNot.name) {
-        val existsNotFun = existsNot.lambda
-        context("non existing") {
-            it("does not throw") {
-                expect(Paths.get("nonExistingFile")).existsNotFun()
-            }
-        }
-
-        val expectedMessageIfExisting = "${NOT_TO.getDefault()}: ${DescriptionPathAssertion.EXIST.getDefault()}"
 
-        context("existing file") {
-            it("throws an AssertionError") {
-                val file = tempFolder.newFile("exists-though-shouldnt")
+            it("${existsNot.name} throws an AssertionError") {
+                val folder = tempFolder.newFolder("exists-though-shouldnt")
                 expect {
-                    expect(file).existsNotFun()
+                    expect(folder).existsNotFun()
                 }.toThrow<AssertionError> {
-                    messageContains(expectedMessageIfExisting)
+                    messageContains("${NOT_TO.getDefault()}: ${DescriptionPathAssertion.EXIST.getDefault()}")
                 }
             }
         }
 
-        context("existing folder") {
-            it("throws an AssertionError") {
-                val folder = tempFolder.newFolder("exists-though-shouldnt")
+        context("file does not exist") {
+            it("${exists.name} throws an AssertionError") {
                 expect {
-                    expect(folder).existsNotFun()
+                    expect(Paths.get("nonExistingFile")).existsFun()
                 }.toThrow<AssertionError> {
-                    messageContains(expectedMessageIfExisting)
+                    messageContains(
+                        "${TO.getDefault()}: ${DescriptionPathAssertion.EXIST.getDefault()}"
+                    )
                 }
             }
+
+            it("${existsNot.name} does not throw") {
+                expect(Paths.get("nonExistingFile")).existsNotFun()
+            }
         }
     }
 
-    describeFun(startsWith.name) {
+    describeFun(startsWith.name, startsNotWith.name) {
         val startsWithFun = startsWith.lambda
+        val startsNotWithFun = startsNotWith.lambda
         context("starts with") {
-            it("does not throw") {
+            it("${startsWith.name} does not throw") {
                 expect(Paths.get("/some/path/for/test"))
                     .startsWithFun(Paths.get("/some/path/"))
             }
-        }
-
-        val expectedMessageIfNotStartsWith = "${DescriptionPathAssertion.STARTS_WITH.getDefault()}:"
 
-        context("does not start with") {
-            it("throws an AssertionError") {
+            it("${startsWith.name} throws an AssertionError") {
+                val expectedMessageIfNotStartsWith = "${DescriptionPathAssertion.STARTS_WITH.getDefault()}:"
```

Comment:
could you please inline expectedMessageIfNotStartsWith, the assertion in line 105 will be more readable this way